### ISSUE TITLE
Trouble parsing negative int64 in PBF

### ISSUE THREAD

**Issue by peterqliu** [mapbox/vector-tile-js#21]
I've been converting some terrain data from PBF to geoJSON, and noticed that contour lines with negative elevations always get converted to a huge value beginning with `184467440737095-----`, and variable numbers for the last 5 digits.

The gist of my conversion process:

```
var tile = new module.exports.VectorTile(new Pbf(pbfContents));

for (var i=0;i<tile.layers.contour.length;i++){
	var feature=tile.layers.contour.feature(i);
	outputgeojson.features.push(feature);
}
```

An example of this output is this [Olympus Mons](https://gist.github.com/peterqliu/544b3ec99b7852383365) contour map, in 500m increments. Most of the volcano is positive elevation, but towards the bottom-left corner, the `-500`,`-1000`, and `-1500` layers get converted to `18446744073709552000`

![screen shot 2015-03-01 at 12 40 49 pm](https://cloud.githubusercontent.com/assets/4551508/6432713/376ab06a-c010-11e4-9f64-87fc67153c15.png)

cc/ @jfirebaugh @mourner

**Comment by yhahn** [comment]
Sounds a bit like https://github.com/mafintosh/protocol-buffers/pull/14 -- we probably need to land something similar in https://github.com/mapbox/pbf?

**Comment by mourner** [comment]
@yhahn hmm, it should work in `pbf` — not sure why this happens, maybe a `vector-tile-js` bug.

**Comment by mourner** [comment]
@peterqliu can you send me a sample pbf file?

**Comment by peterqliu** [comment]
@mourner [PBF](https://b.tiles.mapbox.com/v4/mapbox.mars-terrain-v1/8/72/128.vector.pbf?access_token=pk.eyJ1IjoiaGV5aXRzZ2FycmV0dCIsImEiOiIwdWt5ZlpjIn0.73b7Y47rgFnSD7QCNeS-zA) downloads automatically

the output i get is [this](https://gist.github.com/peterqliu/3db91108b06c89690f3d)

**Comment by mourner** [comment]
@yhahn yeah, you're right — seems like `Pbf`` doesn't handle negative ints encoded as `int` (and not `sint`) correctly. The best solution here would probably be to encode negative values as `sint` rather than `int` in vector tiles — ints are so inefficient... Should be fixed on `Pbf` side as well though.

**Comment by yhahn** [comment]
> The best solution here would probably be to encode negative values as sint rather than int in vector tiles — ints are so inefficient...

Yes definitely, but the past cannot be changed ; )

**Comment by mourner** [comment]
Fixed in 1.1.2. Use `pbf` version >1.3.2 for it to work.
